src/playermat/Playmat.ttslua

```diff
@@ -772,7 +774,7 @@ end
 ---------------------------------------------------------
 
 function drawChaosTokenButton(_, _, isRightClick)
-  Global.call("drawChaosToken", {self, DRAWN_CHAOS_TOKEN_OFFSET, isRightClick})
+  chaosBagApi.drawChaosToken({self, DRAWN_CHAOS_TOKEN_OFFSET, isRightClick})
```

Comment:
Advantage of the Api files is that we don't have to restrict us to a single parameter (and thus make multiple ones into a single table). So'd split these into three separate parameters.

Reply:
Yep, makes perfect sense.  I was trying to just make them behave the same as the original function for consistency, but I agree it's just much more usable to split them all up